security-council-rehearsals/justfile

```diff
@@ -1,13 +0,0 @@
-set dotenv-load
```

Comment:
I assume this is intended here and we don't want also to archive this justfile in the `archive` folder right?

Reply:
Yea I figured I'd just keep the records of the rehearsals completed and delete the source code. We not longer need it.

Reply:
Got it thank you!